discovery_engine_core/core/src/document.rs

```diff
@@ -238,6 +238,20 @@ pub struct HistoricDocument {
     pub title: String,
 }
 
+/// A trending topic.
+pub struct TrendTopic {
```

Comment:
🎨 I'd personally prefer this to be called `TrendingTopic`. I'm assuming `TrendTopic` was chosen because it's shorter, or is there another reason?

Reply:
yes just because it was shorter, but I'd be fine with `TrendingTopic`. perhaps it may change again, when I come to name the more bing-specific type!